Added CSV export

ricferr opened the pull request:
Hello.

How are you ?
A requirement for CSV export popped up so I modified the XML code a bit and added CSV export.

Have a nice weekend,
Ricardo

drmargarido commented:
Hey Ricardo! Very glad to ear from you. Everything is going well on my side :+1: and how are you? 

Thanks for the PR, the submitted code looks nice but it looks like you forgot to send the file `src/exporter/csv.lua`

Also I would ask you to add a command to the cli to also support the csv export there for people who want to integrate d-tracker with other tools. It will allow for calls like:
`d-tracker-cli export-csv -b 3`
`d-tracker-cli export-csv --from 2020-10-05 --to 2020-10-10 `

It can be easily created by doing a copy of this [task](https://github.com/drmargarido/d-tracker/blob/master/src/cli/commands/export_xml.lua), updating the xml things to csv and registering the command in the main file there.

ricferr commented:
Hello.

I'm also fine.

As always I forgot to do the "git add" for the created files :)
I've added the CLI commands. I also fixed a bug that apparently made the CLI exporters always fail because the code was using args.file instead of args.path so it would always error out because args.file would be nil. At least in my system.

drmargarido commented:
Nice catch, thanks!
